src/strings.js

```diff
@@ -1,26 +1,30 @@
-function sayHello (string) {
-  // your code here
-};
+/* eslint-disable prefer-template */
+function sayHello(string) {
+  return 'Hello, ' + string + '!';
```

Comment:
As before, it wants you to use a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) instead of string interpolation. Using template literal, this line would look like:
```
return `Hello, ${string}!`; 
```